internal/testutils/setup.go

```diff
@@ -17,3 +18,11 @@ func SetupTest(t *testing.T) *assert.Assertions {
 	ux.NewUserLog(logging.NoLog{}, io.Discard)
 	return assert.New(t)
 }
+
+func SetupTempTestDir() (string, error) {
+	return os.MkdirTemp(os.TempDir(), "cli-test")
+}
+
+func CleanTempTestDir(testDir string) {
```

Comment:
I am not sure why we need this function, it is just calling another one

Reply:
removed